mobile/examples/object_detection/ios/ORTObjectDetection.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 51;
+	objectVersion = 50;
```

Comment:
FYI - I ran `pod deintegrate` before checking in project.pbxproj which accounts for some of these changes. that's consistent with the other mobile iOS examples.